mp/src/game/client/momentum/c_mom_triggers.cpp

```diff
@@ -525,4 +529,61 @@ IMPLEMENT_CLIENTCLASS_DT(C_TriggerSlide, DT_TriggerSlide, CTriggerSlide)
 RecvPropBool(RECVINFO(m_bStuckOnGround)),
 RecvPropBool(RECVINFO(m_bAllowingJump)),
 RecvPropBool(RECVINFO(m_bDisableGravity)),
-END_RECV_TABLE();
\ No newline at end of file
+END_RECV_TABLE();
+
+// ======================================
+
+LINK_ENTITY_TO_CLASS(info_teleport_destination, C_TeleportDestination);
+
+IMPLEMENT_CLIENTCLASS_DT(C_TeleportDestination, DT_TeleportDestination, CTeleportDestination)
+END_RECV_TABLE();
+
+int C_TeleportDestination::DrawModel(int flags)
+{
+    if (!mom_teledests_draw.GetBool())
+        return 1;
+
+    Color color;
+    MomUtil::GetColorFromHex(mom_teledests_color.GetString(), color);
+    const auto origin = GetAbsOrigin();
+    float dim = mom_teledests_dimensions.GetFloat() / 2.0f;
+
+    CMatRenderContextPtr pRenderContext(materials);
+    const auto pMaterial = materials->FindMaterial(MOM_ZONE_DRAW_MATERIAL, TEXTURE_GROUP_OTHER);
+    IMesh *pMesh = pRenderContext->GetDynamicMesh(true, nullptr, nullptr, pMaterial);
+    CMeshBuilder builder;
+
+    builder.Begin(pMesh, MATERIAL_QUADS, 4);
+
+    builder.Position3fv(Vector(origin.x + dim, origin.y + dim, origin.z).Base());
```

Comment:
These should be `Position3f` without the Vector().Base()

Reply:
This was lost in a rebase somewheres D: